lib/ex_aws/operation/s3.ex

```diff
@@ -34,7 +34,7 @@ defmodule ExAws.Operation.S3 do
 
       headers = headers
       |> Map.put("x-amz-content-sha256", hashed_payload)
-      |> Map.put("content-length", byte_size(body))
+      |> Map.put("content-length", :erlang.iolist_size(body))
```

Comment:
Won't this break backwards compatibility? We resolved this via a [custom `get_byte_size` function](https://github.com/timberio/ex_aws/blob/master/lib/ex_aws/operation/s3.ex#L59-L65)

Reply:
```
iex(1)> :erlang.iolist_size("")
0
```
seems backwards compatible to me, do you have an example where it breaks?

Reply:
Ah, yes, you're correct. I assumed it accepted a list only.